packages/documentation/src/stories/icons/getting-started.stories.mdx

```diff
@@ -19,101 +19,125 @@ import { BADGE } from '@geometricpanda/storybook-addon-badges';
-     ></path>
-   </svg>
-   ```
+### Copy & paste the icons with a postinstall script
 
-3. Be sure to add previously defined styles to each icon (e.g. add the `.post-icon` class to the svg tag).
+Add a `postinstall` script to your `package.json`, to copy & paste the icons to a public folder within your web project.<br/>
+If you want to know more about pre & post scripts and how they are handled, please read the <a href="https://docs.npmjs.com/cli/v7/using-npm/scripts#description" target="_blank">npm documentation</a>.
 
-## Post-Icon component approach
+#### NodeJS version 16.7.0 or newer
 
-With this approach, you use our `<post-icon>` component which allows you to easily embed icons in your HTML code.<br/>
-The component provides properties that let you quickly set where an icon is loaded from (cdn or self-hosted), how it displays and even if it is animated or not. It is also possible to manage a custom icon with our component.
+```json dark
+// package.json
+{
+  "scripts": {
+    "postinstall": "node -e \"const fs = require('fs'); fs.cpSync('node_modules/@swisspost/design-system-icons/public/post-icons', 'public/post-icons', { recursive: true }, (err) => { if (err) throw err; });\""
+  }
+}
+```
 
-### Installation
+#### Older NodeJS versions
 
-To use our icons with the `<post-icon>` component, you do not need to install the **icons package**, because the icons are loaded from the [unpkg CDN](https://unpkg.com/) by default. Instead, you need to install our **components package**.
-Please follow the instructions on the <LinkTo kind="getting-started-components--page">Components - Getting Started</LinkTo> page to do so.
+```json dark
+// package.json
+{
+  "scripts": {
+    "postinstall": "node postinstall-icons.js"
+  }
+}
+```
 
-### Usage
+```js dark
+// postinstall-icons.js
+const fse = require('fs-extra');
 
-Once the component package is installed, you can now simply use our icons in your project as follows:
+const srcDir = 'node_modules/@swisspost/design-system-icons/public/post-icons';
+const destDir = 'public/post-icons';
 
-```html dark
-<post-icon name="1000"></post-icon>
+// To copy a folder or file, select overwrite accordingly
+try {
+  fse.copySync(srcDir, destDir, { overwrite: true });
+  console.log('Icons successfully copied!');
+} catch (err) {
+  console.error(err);
+}
 ```
 
-For more detailed information about the `post-icon` component, continue reading the <LinkTo kind="icons-components-icon--default">post-icon documentation</LinkTo>.<br/>
-A list of all icons is available <LinkTo kind="icons-search-icons--page">here</LinkTo>.
+### Configure the icons base-path
 
-### Self-hosting the icons
+Now that you have the icons available in your project, you need to tell the `icon` component where to find them.<br/>
```

Comment:
```suggestion
Now that you have the icons available in your project, you need to tell the `post-icon` component where to find them.<br/>
```

Reply:
Wontfix, because the name of the component in the docs is `icon`. `post-icon` is only used in the HTML.